src/community/design-system-day/speaker-information.md

```diff
@@ -0,0 +1,175 @@
+      },
+      value: {
+        text: "10 and 11 October 2023"
+      }
+    },
+    {
+      key: {
+        text: "Location"
+      },
+      value: {
+        html: '<a href="https://dynamicearth.org.uk/plan-your-visit/getting-here/">Dynamic Earth</a>, Edinburgh, and online'
+      }
+    }
+  ]
+}) }}
+
+## Speakers 
+
+Our speakers come from organisations across the UK and international public sector. We're joined by folks working on design systems from notable companies too.
+
+## Day 1 speakers
+
+### Kelly Lee (GOV.UK Design System team)
+
+Kelly is a delivery manager on the GOV.UK Design System team. Before joining the Government Digital Service, she worked at Jobcentre Plus in a variety of roles including business coach, performance team leader and financial assessor.
+
+### Imran Hussain (GOV.UK Design System team)
+Imran is a community consultant with over 10 years experience building communities across different sectors. He believes communities can be a force for good in society and loves working with communities that have a positive impact on public life.
+
+He is currently a Community Designer at Government Digital Service, a department in the UK Government. He works on creating the right conditions for a community to thrive around the ever-popular GOV.UK Design System.
+
+### Rachel Male (Scottish Governement)
+Rachel is User Researcher and works in the Social Security Directorate for the Scottish Government. Rachel helps design disability and low income benefits for the people of Scotland and currently leads research operations focusing on operationalising their user research. Rachel has a passion for ensuring user research is effective, efficient and informative.
+
+### Samantha Ernstzen (Scottish Governement)
+To be confirmed
+
+### Jennifer I'Anson (Scottish Governement)
+To be confirmed
+
+### GOV.UK Design System team 
+The <a href="/design-system-team/" class="govuk-link">GOV.UK Design System team</a> is responsible for the implementation and maintenance of styles, components and patterns on the GOV.UK Design System. 
+
+### Deborah Dada (Storelab)
+Deborah is a Product Designer with a strong focus on crafting meaningful user experiences and simplifying complex business solutions. Deborah brings a wealth of hands-on experience in UX principles and creating design systems with expertise that spans research, design systems and success metrics, underpinned by a design thinking approach. Deborah is currently a Product Designer at StoreLab and contributes to enhancing user satisfaction.
+
+### Caroline Jarrett (Effortmark)
+Caroline is the Forms Specialist. She advises organisations on how to make forms easier, improve websites and business processes. Her work spans all the types of design that will improve forms. Caroline is a sought-after speaker, trainer and author. Her latest book is 'Surveys that work: a practical guide for designing and running better surveys'.
+
+### Frankie Roberto (Department for Education)
```

Comment:
Sorry - having seen the other bios I realised I should make it clear that I'm there as a freelancer not representing the department I’m contracted to.

```suggestion
### Frankie Roberto (Independent)
```